web-dev/src/App.js

```diff
@@ -48,7 +48,7 @@ function App() {
                      element={<LoginPage />} />
               <Route path="registration"
                      element={<RegistrationPage />} />
-              <Route path="profile/edit"
+              <Route path="registration/edit"
```

Comment:
This is tricky. We already use "profile/:username" to handle all other user profile stuff. Edit profile "profile/edit" might jump into that category.

Reply:
I will do profile-edit then.